<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBarModule</class>
 <widget class="QWidget" name="ToolBarModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="6">
    <widget class="QComboBox" name="sortingBox">
     <property name="toolTip">
      <string>Sort By</string>
     </property>
     <item>
      <property name="text">
       <string>A-Z</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Z-A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Size</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Type</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>First Modified</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last Modified</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="creatingButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Create</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resources/resources.qrc">
       <normaloff>:/fileSystemIcons/fileSystemIcons/new-document.png</normaloff>:/fileSystemIcons/fileSystemIcons/new-document.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="hiddenButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Show Hidden</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resources/resources.qrc">
       <normaloff>:/fileSystemIcons/fileSystemIcons/hidden.png</normaloff>:/fileSystemIcons/fileSystemIcons/hidden.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="replacingButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Replace</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resources/resources.qrc">
       <normaloff>:/fileSystemIcons/fileSystemIcons/import-export.png</normaloff>:/fileSystemIcons/fileSystemIcons/import-export.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>399</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="removingButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Remove</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resources/resources.qrc">
       <normaloff>:/fileSystemIcons/fileSystemIcons/bin.png</normaloff>:/fileSystemIcons/fileSystemIcons/bin.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="renamingButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Rename</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resources/resources.qrc">
       <normaloff>:/fileSystemIcons/fileSystemIcons/rename (1).png</normaloff>:/fileSystemIcons/fileSystemIcons/rename (1).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="copyingButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Copy</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resources/resources.qrc">
       <normaloff>:/fileSystemIcons/fileSystemIcons/copy.png</normaloff>:/fileSystemIcons/fileSystemIcons/copy.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="8">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/resources.qrc"/>
  <include location="../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
